docs/sources/panels-visualizations/configure-standard-options/index.md

```diff
@@ -25,84 +25,94 @@ weight: 60
 
 # Configure standard options
 
-The data model used in Grafana, namely the [data frame](https://grafana.com/developers/plugin-tools/introduction/data-frames), is a columnar-oriented table structure that unifies both time series and table query results. Each column within this structure is called a _field_. A field can represent a single time series or table column.
+**Standard options** in the panel editor pane let you change how field data is displayed in your visualizations. Options that you apply don't change the data, they just change how Grafana _displays_ the data.
```

Comment:
Moving away from a technical description of data frames here seems like a very good thing 😄 🥳

Reply:
Big shoutout to @baldm0mma for his help here!